scripts/mempool.js

```diff
@@ -175,6 +182,7 @@ export class Mempool {
                     );
                     if (!isMyUTXO && !this.isSpent(op)) {
                         this.spent.set(tx.txid, op);
+                        if (wallet.isCoinLocked(op)) wallet.unlockCoin(op);
```

Comment:
Nit: Make a `setSpent` method to avoid having to remember to unlock the coin if spent (Similar to how we have a `isSpent` method